mlflow/server/__init__.py

```diff
@@ -378,4 +388,28 @@ def _run_server(
     else:
         # This shouldn't happen given the logic in CLI, but handle it just in case
         raise MlflowException("No server configuration specified.")
-    _exec_cmd(full_command, extra_env=env_map, capture_output=False)
+
+    # The `HUEY_STORAGE_PATH_ENV_VAR` is used by both Mlflow server handler workers and huey job runner (huey_consumer).
```

Comment:
@WeichenXu123 can we add a comment to the code explaining why it's okay to put the huey database in a temp directory? (Because we store job definitions in our own database, so Huey tasks don't need to persist?)